Toolkit/ArcGISToolkit/AR/ArcGISARView.swift

```diff
@@ -217,10 +198,8 @@ public class ArcGISARView: UIView {
             return
         }
         
-        if let origin = originCamera {
-            // We have a starting camera.
-            initialTransformationMatrix = origin.transformationMatrix
-            sceneView.setViewpointCamera(origin)
+        if let _ = originCamera {
```

Comment:
I remember reading somewhere that this is the generally preferred way of checking for `nil`:

```suggestion
        if originCamera != nil {
```